CHANGELOG.md

```diff
@@ -1,37 +1,73 @@
+- This feature release requires a **minimum Go version of v1.19.7 or higher to successfully build Lotus.** Go version v1.20 and higher is supported.
+- *Storage Providers:* The proofs libraries now have CUDA enabled by default, which requires you to install [CUDA](https://lotus.filecoin.io/tutorials/lotus-miner/cuda/) if you haven't already done so. If you prefer to use OpenCL on your GPUs instead, check out the documentation [here](https://lotus.filecoin.io/kb/using-opencl/#using-opencl-instead-of-cuda)
+- Please also familiarize yourself with the [changelog in v1.23.0](https://github.com/filecoin-project/lotus/releases/tag/v1.23.0)
 
-- feat: daemon: Auto-resume interrupted snapshot imports ([filecoin-project/lotus#10636](https://github.com/filecoin-project/lotus/pull/10636))
+If you are upgrading from Lotus v1.23.0 the above warnings should be familiar to you. One thing to note when going from v1.23.0 to v1.23.1 is that the Lotus-Miner legacy-markets has now been disbled by default and will be removed in the near term future. Users are adviced to migrate to [Boost](https://boost.filecoin.io) or other SP-market systems.
+
+## Highlights
+
+**🛣 Execution Lanes 🛣**
+This feature release introduces VM Execution Lanes! Execution lanes efficiently divide the workload between system processes  (chainsync) and RPC requests. This way syncing the chain will not be at the mercy of responding to users' requests and RPC providers nodes should have less problems catching up.
+
+To take advantage of VM Execution Lanes, you need to set up two environment variables:
+- `LOTUS_FVM_CONCURRENCY` - read more abot what this value should be set to here:
+- `LOTUS_FVM_CONCURRENCY_RESERVED = 4` 
+
+**🧱 Aggregation / Batching fixes 🔨**
+
+Numerous aggregation and batching fixes has been included in the feature release. Large `ProveCommitAggregate` and `PreCommitBatching` messages that exceeds the block limit will now automatically be split into smaller messages when sent to the chain.
+
+Additionally we have added a new feature that staggers the amount of ProveCommit messages sent simulatanously to the chain if a storage provider has been aggregating many sectors in ProveCommitAggregate message, but at the time of publishing the BaseFee is below the aggregation threshold. This stagger feature prevents issues where some of the ProveCommit messages fail with the SysErrorOutOfGas message. You can tweak how many messages will be staggered per epoch by changing `MaxSectorProveCommitsSubmittedPerEpoch` in the [sealing section of the config.toml file.](https://lotus.filecoin.io/storage-providers/advanced-configurations/sealing/#sealing-section)
```

Comment:
add FIL+ gas cost call out

Reply:
Added call out about the FIL+ gas cost when aggregating ProveCommit messages here: https://github.com/filecoin-project/lotus/pull/10990/commits/bd22319418b3485932aab97aa60255d11370e87f